src/lib/seo/index.ts

```diff
@@ -9,19 +9,19 @@ export type PageUrl = {
   pathname: string,
 };
 
-export const siteName = () => {
+/** 
+  * `globalSeo` _should_ have a key per available locale. When there is only one
+  * locale configured in Dato, that key is missing. Therefore we fallback to the
+  * `globalSeo` object 
+  */
+const localeSeo = () => {
   const locale = getLocale();
-  const localeSeo = globalSeo[locale as keyof typeof globalSeo];
-
-  return localeSeo.siteName;
+  const localeSeoData = globalSeo[locale as keyof typeof globalSeo];
+  return localeSeoData || globalSeo;
```

Comment:
Yes this is nitpicky 😅, but I find variables that kind of hold the same value, and then differ only slightly in name quite confusing (ie. `localeSeo` and `localeSeoData`). 

```suggestion
  return globalSeo[locale as keyof typeof globalSeo] || globalSeo;
```

Reply:
I understand the confusion between variable names that differ slightly, however, my original line 19 was exactly that, the _locale_ version of the seo data. and it differs from the globalSeo conceptually.

the line `return localeSeoData || globalSeo` tells me the _intent_ of the code: use the locale version first, fallback to the global when not available. Which is what this function is all about. Besides, my localeSeoData name explains the (quite confusing) type magic.

Your suggestion does not convey the intent or the typescript magic.

What do you think?